index.js

```diff
@@ -727,27 +730,40 @@ var tick = function() {
 				if(!("units" in node)) {
 					node.units = {};
 				}
-				if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2612211153/issues/source?bid=3248399&fileBranchId=3307983#l733)